iceberg-theme/static/css/iceberg-theme.css

```diff
@@ -241,32 +250,86 @@ h4:hover a { visibility: visible}
     display: grid;
     grid-template-columns: 5fr 1fr;
     grid-gap: 1rem;
+    height: 100vh;
+    width: 100vw;
+    padding-top: 1rem;
+    overflow-y: scroll;
+}
+
+div.sidebar {
+    text-decoration: none;
+    left: 0;
+    top: 0;
+    overflow-y: scroll;
+    background-color:#FFF;
+    display: block;
+    font-size: 16px;
+    min-width: 15rem;
+    list-style-type: none;
+}
+
+.sidebar ul {
+    text-decoration: none;
+    list-style: none;
+}
+
+.sidebar li a {
+    color: black;
+}
+
+.sidebar li a:hover {
+    color: #2879BE;
+}
+
+#active {
+    color: #2879BE;
+}
+
+i.fa.fa-chevron-right {
+ float: right;
+}
+
+i.fa.fa-chevron-down {
+ float: right;

```

Comment:
This isn't a major concern but I would steer clear of `floats` if possible and use flexbox instead. `display: flex; justify-content: flex-end;`

Reply:
Unless you are overriding existing floats from the theme and in that case, you are kind of forced to use override floats.

Reply:
Changed these to flexbox!